lasp_opensearch_data_center/constructs/frontend.py

```diff
@@ -121,25 +122,30 @@ def __init__(
             }
         )
 
-        # Define CI/CD group and policy names
-        frontend_group_name = f"{account_type}FrontEndGroup"
-        frontend_policy_name = f"{account_type}-frontend-automated-deploy"
+        # Define CI/CD user, role, and policy names
+        frontend_role_name = "jenkins-frontend-deploy-role"
```

Comment:
I suggest we leave all these names and IDs as generic as possible. e.g. "frontend-deploy-role"

Reply:
Sounds good, simple fix.